authoring-new-actions.md

```diff
@@ -0,0 +1,130 @@
+  8. **Mark Initial repo visibility as**  **PRIVATE**
+  9. **Add** [actions-admin](https://repos.opensource.microsoft.com/Azure/teams/actions-admin) **as one of the**  **Administrators Teams who could manage repo settings. You can also add more teams if needed.**
+  10. **Choose License template (Generally all actions are open sourced under MIT license)**
+  11. **Review all the submitted details and create Request.**
+5. Note that this request triggers a Legal OSS approval process which needs to be completed before the repo visibility can be changed to PUBLIC. You should get an email from &quot;[noreply-opensource@microsoft.com](mailto:noreply-opensource@microsoft.com)&quot;  with a link to the work item that tracks the legal, OSS and your LT clearance.
+
+## Guidelines during Development phase
+
+- Ensure that no sensitive information is leaked in logs
+- Do Not use the action repo for creating any workflows for testing as it leaves the history of workflow logs under the tab: Actions. Please create a separate repo for test workflows.
+- While the Action repo is still private, you cannot access it from workflows in other repos. Hence testing on official repo will be possible only after you make the repo public and tag the code with a V0-BETA version. But do not publish the action to Marketplace.
+- To enable debugging in the workflow logs, set the secret: ACTIONS\_STEP\_DEBUG = true
+
+## ReadMe.md &amp; action.yml files
+
+Once the Action implementation is complete, ensure that it is documented well in the readme.md file at the root of the repo of the Master branch. Please note that this content is used when the Action is published to the GitHub Marketplace. Hence including a sample workflow which covers the best usage of the Action is always recommended.
+
+Also ensure that the Action.yml is present in the root of the repo and has the Action name &amp; description well defined. Also choose the icon &amp; color suitably under branding as these are used when the Action is listed in Marketplace catalog.
+
+Note: All actions published from **Azure** GitHub org are branded using the Azure icon irrespective of the icon used in the action.yml. This is to leverage the recall value of Azure brand and icon as compared to the individual icons of various Azure services. ![](RackMultipart20201207-4-o60y8a_html_71d21847106d5771.png)
+
+## Action Versioning
+
+Versioning guidance for GH action that should be adhered to before an action is released:
+
+URL: [https://github.com/actions/toolkit/blob/master/docs/action-versioning.md](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Factions%2Ftoolkit%2Fblob%2Fmaster%2Fdocs%2Faction-versioning.md&amp;data=02%7C01%7Cushan%40microsoft.com%7C8aae17deba8746e1a01c08d758875665%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637075210643253612&amp;sdata=%2BePOVjbQc4bH1FYUlbCFx%2BNCjbfXSGd%2F2P96ncGWaII%3D&amp;reserved=0)
```

Comment:
You could just use `URL: https://github.com/actions/toolkit/blob/master/docs/action-versioning.md` The URLs get highlighted by default